config/config.exs

```diff
@@ -68,6 +68,13 @@ config :wraft_doc, Oban,
      ]}
   ]
 
+config :ex_typesense,
```

Comment:
Lets move all they keys we need to fetch from variables to `runtime.exs`



https://hexdocs.pm/elixir/config-and-releases.html#configuration - Read this section for more details